app/lib/gh-docs/docs.ts

```diff
@@ -81,19 +82,22 @@ function parseAttrs(
  * let's have simpler and faster deployments with just one origin server, but
  * still distribute the documents across the CDN.
  */
-global.docCache ??= new LRUCache<string, Doc>({
-  // let docCache = new LRUCache<string, Doc | undefined>({
+global.docCache ??= new LRUCache<string, Doc, CacheContext>({
   max: 300,
   ttl: NO_CACHE ? 1 : 1000 * 60 * 5, // 5 minutes
   allowStale: !NO_CACHE,
   noDeleteOnFetchRejection: true,
   fetchMethod: fetchDoc,
 });
 
-async function fetchDoc(key: string): Promise<Doc> {
+async function fetchDoc(
+  key: string,
+  _staleValue: Doc | undefined,
```

Comment:
I'm not sure I understand why `_staleValue` is in here, doesn't seem like we use it. Also, when we call `docCache.fetch` we're not passing it in, am I missing something?

Reply:
it’s a required argument when using the LRU.FetchOptionsWithContext as we’re passing around an octokit instance, but yes, we’re not using it, it’s added by LRUCache. followed convention from other cache.fetch functions

Reply:
Great, figured I was missing something, thanks for the explanation!